File: Unit_2_C_Programming/7_Pre-Processor Directives/Lecture_Labs/Lab_1/interpreter/lab1.py
"""
======================================================================================================================
 Name        : Lab_1
 Author      : Eng. Mohamed Kadry Hussein
 Created on  : 21/8/2024
 Description : Unit 2 C-Progtamming >>Pre-Processor Directives>>Lab_1
 ======================================================================================================================
"""
import datetime as dt

# Record the start time
start_time = dt.datetime.now()

# Simulate user input for two numbers
print("Enter your first number: ")
# num1 = int(input(":"))
print("Enter your second number: ")
# num2 = int(input(":"))
num1 = 1
num2 = 2

# Calculate the sum of the two numbers
result = num1 + num2
print(f"The sum is: {result}")

# Calculate the elapsed time
elapsed_time = dt.datetime.now() - start_time

# Display the elapsed time in seconds and microseconds
print(f"Time taken: {elapsed_time.seconds} seconds and {elapsed_time.microseconds} microseconds")

File: Unit_2_C_Programming/3_Array_&_String/Lecture_Labs/lab_1.c
/*
 ======================================================================================================================
 Name        : lab_1_scan_and_print_students_degrees
 Author      : Eng.Mohamed Kadry Hussien
 Created on	 : 31/7/2024
 Description : Unit 2 C-Programming >>Array&string >> lab1
 ======================================================================================================================
 */

#include <stdio.h>
#include <stdlib.h>

int main()
{
    float degrees[10];

    for(int i=0;i<=9;i++)
    {
        printf("Enter the student %d degree :",i+1);
        scanf("%f",&degrees[i]);
    }

    for(int i=0;i<=9;i++)
    {
        printf("the student %d degree is :%f\n",i+1,degrees[i]);
    }
    return 0;
}
